Maui.RevenueCat.InAppBilling/Platforms/iOS/Extensions/EntitlementInfosExtensions.cs

```diff
@@ -0,0 +1,36 @@
+﻿using Maui.RevenueCat.InAppBilling.Models;
+using Maui.RevenueCat.iOS;
+
+namespace Maui.RevenueCat.InAppBilling.Platforms.iOS.Extensions;
+internal static class EntitlementInfosExtensions
+{
+    internal static List<EntitlementInfoDto> ToEntitlementInfoDtoList(this RCEntitlementInfos entitlements)
+    {
+        var entitlementInfoDtos = new List<EntitlementInfoDto>();
+
+        foreach (var entitlement in entitlements.All.Values)
+        {
+            var entitlementInfoDto = new EntitlementInfoDto()
+            {
+                BillingIssueDetectedAt = entitlement.BillingIssueDetectedAt?.ToDateTime(),
+                ExpirationDate = entitlement.ExpirationDate.ToDateTime(),
+                Identifier = entitlement.Identifier,
+                IsActive = entitlement.IsActive,
+                IsSandbox = entitlement.IsSandbox,
+                LatestPurchaseDate = entitlement.LatestPurchaseDate.ToDateTime(),
+                OriginalPurchaseDate = entitlement.OriginalPurchaseDate.ToDateTime(),
+                OwnershipType = entitlement.OwnershipType.ToOwnershipId(),
+                PeriodType = entitlement.PeriodType.ToPeriodId(),
+                ProductIdentifier = entitlement.ProductIdentifier,
+                ProductPlanIdentifier = string.Empty, // Does not exist on ios???
```

Comment:
You are correct, it doesn't.
https://revenuecat.github.io/purchases-ios-docs/4.27.2/documentation/revenuecat/entitlementinfo

Please change the comment to `It doesn't exist on iOS`